src/commands/force/mdapi/deploy/report.ts

```diff
@@ -68,7 +71,7 @@ export class Report extends DeployCommand {
         : new DeployProgressStatusFormatter(this.logger, this.ux);
       progressFormatter.progress(deploy);
     }
-    this.deployResult = await deploy.pollStatus(500, this.getFlag<Duration>('wait').seconds);
+    this.deployResult = await deploy.pollStatus(500, waitDuration.seconds);
```

Comment:
Do you want to use the polling options object here too?

Reply:
yeah, lemme fix that, too